emrun.py

```diff
@@ -707,10 +716,21 @@ def do_POST(self):
         have_received_messages = True
       elif data.startswith('^exit^'):
         if not emrun_options.serve_after_exit:
-          page_exit_code = int(data[6:])
+          # Emrun callers
+          try:
+            # Treat sending 'undefined' (i.e. EXITSTATUS = undefined) as exit code zero...
+            page_exit_code = (0 if data == '^exit^undefined' else int(data[6:]))
```

Comment:
Its seems odd to me that this should be possible. 

When can it happen that `^exit^` is triggered without an exit status at all?   Should we treat this instead as an error?   Or as `Unexpected page exit result` perhaps?